src/main/java/Blue/Input.java

```diff
@@ -0,0 +1,100 @@
+package Blue;
+
+/**
+ * Defines an encapsulation of commands and command arguments well understood by Blue.
+ */
+public class Input {
+    private InputCommand command;
+    private int taskIndex;
+    private String taskQuery;
+    private Task taskToAdd;
+    private String errorMessage;
+
+    /**
+     * Class level constructor for ill-defined inputs.
```

Comment:
what is class-level constructor?

Reply:
I see this is not a naming convention, corrected.